### ISSUE TITLE
Some notes on Windows don't display anything when opened

### ISSUE THREAD

**Issue by erik-brangs** [tomboy-notes/tomboy-ng#65]
I've got a note on Windows that can't be opened properly. When opening it, the window contains neither the note title nor the note text. When trying to click the delete button, I immediately get the message about data corruption. Pressing abort leads to the following trace:

```
TApplication.HandleException Access violation
  Stack trace:
  $00000001002017E8
  $0000000100200089
  $000000010012F7A8
  $000000010015023A
  $000000010015096F
  $000000010013C4A1
  $0000000100150102
  $000000010000EDA5
  $000000010012E7B1
  $0000000100122ADC
  $00000001001A7527
  $00000001001052B9
  $0000000100105A7C
  $00000001001EEA0E
  $00007FF800ED6CC1
  $00007FF800ED62CE
  $00007FF800ED6058
WARNING: TBitBtn.Destroy with LCLRefCount>0. Hint: Maybe the component is processing an event?
```

I deleted the debug log before triggering the problem. It doesn't contain anything else other than the trace.

**Comment by davidbannon** [comment]
I am afraid the stack trace does not really help. I am assuming you are using 0.16 of tomboy-ng ?  Would it be possible for you to send me this note if it has nothing private in it ?

Otherwise, trick might be for me to send you a version of tomboy-ng binary with full debug enabled. Its pretty wordy and slow but might tell us whats happening.

Windows is not the easiest platform to debug from with its no console issues but I really would like to track this down if at all possible. I thought I had all bad xml issues suitably handled and reported on.

Ah, I wonder if you know the note title ?   Perhaps not. I think a special debug version might be best approach. I could add a --debug-index option for this purpose.....

Davo

**Comment by davidbannon** [comment]
This problem was caused by a particular cunning xml problem. A note with a title in XML but not in content (a rather strange aspect of the Tomboy schema).  It can now. I have released some new binaries (v0.16a) and will do remainder in a few days.
